com.microsoft.mrtk.input/Simulation/Prefabs/MRTKInputSimulator.prefab

```diff
@@ -68,6 +68,10 @@ MonoBehaviour:
     simulationMode: 1
     anchorPoint: 0
     changeNeutralPose: {fileID: -1599196918749775551, guid: 70b361cf3c037d54896482d1ffd9f999, type: 3}
+    defaultPose: 2
+    selectionPose: 3
+    secondaryDefaultPose: 1
+    secondarySelectionPose: 7
```

Comment:
I wonder if we want to use a different term than "selection" pose here. I see that the `secondarySelectionPose` is set to `ThumbsUp`, which I don't think will trigger a selection in any of our existing interactables, right? Maybe "action pose" or something?

Reply:
I'm down for renaming, but am unsure what to rename it to. I defaulted to calling it a "selection" pose only because triggering it is bound to the select action.

in MRTK2 we had Left Mouse Button Pose, Right Mouse Button Pose, and Middle Mouse Button Pose, but semantically that doesn't quite work here because of how it's all tied to input actions.

Reply:
I'll rename these actions to Trigger Button Pose because that's what the input action binding is called in our controls